adapters/amqp-vertx/src/test/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapterTest.java

```diff
@@ -158,25 +161,43 @@ public void testStartUsesClientProvidedAmqpServer(final TestContext ctx) {
     }
 
     /**
-     * Verifies that the AMQP Adapter does not support anonymous relay.
+     * Verifies that the AMQP Adapter support anonymous relay both for authenticated and unauthenticated clients.
      */
     @Test
-    public void testAnonymousRelayNotSupported() {
+    public void testAnonymousRelaySupportedForAllClients() {
```

Comment:
IMHO this should be `testAnonymousRelayRequired` and verify that a client cannot open a link that contains a target address